app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer_test.go

```diff
@@ -345,3 +346,71 @@ func TestDeleteLoadBalancer(t *testing.T) {
 		t.Errorf("unexpected health checks have not been deleted: %v", fhc.EnsuredHealthChecks)
 	}
 }
+
+func TestFormatLoadBalancersList(t *testing.T) {
+	testCases := []struct {
+		statuses       []status.LoadBalancerStatus
+		expectedOutput string
+	}{
+		{
+			// No ingresses.
+			[]status.LoadBalancerStatus{},
+			"No multicluster ingresses found.",
+		},
+		{
+			// Name only.
+			[]status.LoadBalancerStatus{
+				status.LoadBalancerStatus{
+					LoadBalancerName: "lb",
+				},
+			},
+			"NAME      IP        CLUSTERS\nlb                  \n",
```

Comment:
would this be easier to encode with "\t" instead of literal tabs?

Reply:
Yeah, I wonder the same. It might be more readable to use `\t`

Reply:
The spacing is not the same in every case and its not always "\t". Depends on the length of value. It gets confusing with lots of /t's and spaces.